src/Sylius/Bundle/CoreBundle/Resources/config/services/installer.xml

```diff
@@ -12,6 +12,9 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="currency">USD</parameter>
```

Comment:
```suggestion
        <parameter key="sylius_installer_currency">USD</parameter>
```

Reply:
I guess this means also changing the name of the argument on L36?

Reply:
I would say so. `Locale` is a global parameter for the whole app. Here we are using it only for installation purpose. Could you also add a mention about it somewhere to docs?

Reply:
Sure can!